File: scraper/scrape.js
var stock = phantom.args[0];
var page = require('webpage').create(),
    url = 'http://www.bvb.ro/ListedCompanies/SecurityDetail.aspx?s=' + stock;

page.open(url, function (status) {
    if (status !== 'success') {
        console.log('Unable to access network');
    } else {
        var results = page.evaluate(function() {
	    var res=[];	
            var list = document.querySelectorAll('#ctl00_central_lbPrice');
            for(var i=0; i < list.length;++i){
		 res.push(list[i].innerText);
	    }
            return res;
        });
  
    }
    console.log(results);
    phantom.exit();
});

File: scraper/scrape.py
#!/usr/bin/env python

from BeautifulSoup import BeautifulSoup
import urllib2

url="http://bvb.ro/ListedCompanies/SecurityDetail.aspx?s=FP"
page=urllib2.urlopen(url)
soup = BeautifulSoup(page.read())
stocks=soup.findAll("span", {"id": "ctl00_central_lbPrice"})
for stock in stocks:
	print stock.text
